openlibrary/catalog/get_ia.py

```diff
@@ -26,16 +26,17 @@ class NoMARCXML(IOError):
     pass
 
 
-def urlopen_keep_trying(url):
+# TODO: there are few other modules that call this, refactor to rename later to request_keep_trying
```

Comment:
Which other modules?  If we land this PR, do we break those other files?

Reply:
Sorry - it is really just openlibrary/catalog/marc/marc_subject.py, which I included in this PR.  I'll change this comment to note that and remove the TODO.